### PR TITLE
Update links

### PR DESCRIPTION
**Pull request by klust**
I don't know if it is useful to update links in pages from the past, but this pull request updates a number of web links in all versions and corrects a lot of internal links in the LUST tutorial.

### CODE REVIEW

File: docs/2020-06-isc20/contributing.md [hunk 1]
```diff
@@ -83,13 +83,13 @@ a project that aims to *automate* tedious software installation procedures.
 
 At the end of 2015 efforts were made to tackle this issue by implementing
 GitHub integration features in EasyBuild, which automate the contribution
-workflow by running `git` commands and interacting with the [GitHub API](https://developer.github.com/v3/).
+workflow by running `git` commands and interacting with the [GitHub API](https://blogs.oracle.com/linux/post/attack-of-the-cosmic-rays).
```
Comment (boegel) [inline docs/2020-06-isc20/contributing.md:86]: @klust This replacement in faulty? (it was done correctly in `docs/2021-isc21/contributing.md` though)

File: docs/2021-lust/github_integration.md [hunk 2]
```diff
@@ -77,13 +77,13 @@ a project that aims to *automate* tedious software installation procedures.
 
 At the end of 2015 efforts were made to tackle this issue by implementing
 GitHub integration features in EasyBuild, which automate the contribution
-workflow by running `git` commands and interacting with the [GitHub API](https://developer.github.com/v3/).
+workflow by running `git` commands and interacting with the [GitHub API](https://blogs.oracle.com/linux/post/attack-of-the-cosmic-rays).
```
Comment (boegel) [inline docs/2021-lust/github_integration.md:80]: @klust Also wrong replacement here

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by klust** [comment]
Sorry, should have done this sooner, but I was too busy with the LUMI course last week.

File: docs/2020-06-isc20/contributing.md (hunk 1 above)
Comment (klust) [reply docs/2020-06-isc20/contributing.md]: Indeed faulty and corrected in a new commit. That's what happens when you switch all the time between normal computers and a Mac with a different key combination for copy and paste and update a lot of links one after another...

File: docs/2021-lust/github_integration.md (hunk 2 above)
Comment (klust) [reply docs/2021-lust/github_integration.md]: Should also be corrected in the latest commit.

**Review by boegel — APPROVED** [review]

**Pull request merged**